requirements/requirements.txt

```diff
@@ -3,4 +3,5 @@ scikit-learn
 mdlearn
 matplotlib
 plotly
+biopython
 git+https://github.com/ramanathanlab/gene_transformer
```

Comment:
@braceal So the way we have the gene_transformer repo set up now - will this line in the requirements handle the dependencies for the gene_transformer repo or would those need to get manually installed like in the gene_transformer README?

Reply:
They would need to get manually installed I believe.